### DISCUSSION TITLE
Build with Beckn - ONEST Integration Guide

### DISCUSSION THREAD

**Discussion by archit15singh** [ONEST-Network/community#26] [Show and tell]
🚀 New Video Alert: Build with Beckn - ONEST Integration Guide! 🚀

Hey Beckn Developers! 🌟

We're excited to share our latest tutorial: "Build with Beckn: Step-By-Step Developer's Guide to ONEST Integration". This comprehensive video is a must-watch for anyone looking to learn ONEST integration with Beckn protocols.

🔗 Watch Now: https://www.youtube.com/watch?v=sDJpofBFXiM

What You'll Learn:

Beckn overview and use cases
Understanding the Beckn-ONEST integration process (BAP setup)
Step-by-step guidance on setup and configuration

Don't miss out on this opportunity to level up your Beckn development skills. Check out the video, and let's build something amazing together!

Happy Coding! 💻

**Comment by amankashyap697** [comment]
Hey Archit,
Watched your video and it must help but the git protocol-server repo is unavailable 
[https://github.com/beckn/protocol-server/tree/master](url).

Can u please help?
